cmd/goQuery/cmd/root.go

```diff
@@ -204,18 +205,8 @@ func initLogger() {
 	}
 	if cmdLineParams.Format == "json" {
 		format = logging.EncodingJSON
-
-		// if there is a query log, write log lines to that
-		queryLog := viper.GetString(conf.QueryLog)
```

Comment:
This is removing the tight link to the query log. Which was bonkers to begin with.